.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You added this so linting is enforced?
It kind of broke my unit testing through the VSCode plugin, maybe we could include those settings here also?

Reply:
Yes. Merge that in as well.

Reply:
I will add it.

Reply:
I enabled `"python.testing.pytestEnabled": true` since we're using pytest now.